modules/microshift-mirror-container-images.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mirror-container-images_{context}"]
+= Mirror {product-title} container images into an existing registry
+
+Mirroring container images is an alternative to embedding {product-title} into a {op-system-ostree-first} image. Container images and updates can be transferred to air-gapped environments and installed on {product-title} instance. But, using a custom air-gapped container registry can be necessary with certain user environments and workload requirements.
```

Comment:
Mirroring is not an alternative to embedding without the air-gapped context. Can we keep the wording closer to the https://docs.google.com/document/d/1aqgjxP2YqIGi2djU96vN2yykDZpIC8s2hg4QHTCz390/edit#heading=h.ykqn5kc26aev?